src/contexts/sendSearchRequest.ts

```diff
@@ -86,9 +86,12 @@ export const sendSearchRequest = async ({
           ? {
               reranking_config: {
                 reranker_id: rerankerId,
-                mmr_config: {
-                  diversity_bias: rerankDiversityBias,
-                }
+                ...(rerankerId === 272725718 ? {
```

Comment:
Perhaps extract `272725718` into a variable called `DIVERSITY_RERANKER_ID` that can be imported and used anywhere. That way we have less risk of a typo or other mistake messing this up someday.

Reply:
Great suggestion. Done!